src/docs/release/breaking-changes/target-platform-linux-windows.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Adding `linux` and `windows` to [`TargetPlatform`][] enum
+description: Two new values were added to the [`TargetPlatform`][] enum that could require additional cases in switch statements that switch on a [`TargetPlatform`][].
```

Comment:
The meta data shouldn't have any links. Remove both of these

Reply:
OK, done.  Can the metadata lines be wrapped, or do they have to be one per line?

Reply:
Just seeing this question now! The `description:` line in the YML meta data needs to be a single line or Travis fails.